README.md

```diff
@@ -1,4 +1,6 @@
 # Personal-Website-Part-1
 Building a Personal Website part 1.
 
-Deployment link: https://eddie1116.github.io/Personal-Website-Part-1/
\ No newline at end of file
+Deployment link: https://eddie1116.github.io/Personal-Website-Part-1/
+
+Working on each steps one day at a time used example on block 9 to create my P.W.S
```

Comment:
Using the example in block 9 to work on my project website, I will make changes as time goes on and learn from what I am doing one day at a time. My project will not reflect what is on block 9 it is merely a guide for me to use moving forward. Will make changes as time goes on.